### PR TITLE
Add metrics-server presubmits

### PR DESCRIPTION
**Pull request by serathius**
/cc @s-urbaniak 
/cc @brancz
Refs https://github.com/kubernetes-sigs/metrics-server/issues/483

### CODE REVIEW

**Comment by serathius** [comment]
ping @krzyzacy

File: config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  - name: pull-metrics-server-test-unit
+    decorate: true
+    path_alias: sigs.k8s.io/metrics-server
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200312-a0f211f-master
+        args:
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml:26]: here should be `command`, like other jobs above.

File: config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    skip_branches:
+      - release-0.3
+    path_alias: sigs.k8s.io/metrics-server
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200312-a0f211f-master
+        args:
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml:74]: (ditto)

File: config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml (hunk 1 above)
Comment (serathius) [reply config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml]: done

File: config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml (hunk 2 above)
Comment (serathius) [reply config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml]: done

**Pull request merged**